jobs/checkAssignedTicketResponseTimes.ts

```
@@ -65,29 +77,29 @@ export const checkAssignedTicketResponseTimes = async (now = new Date()) => {
-              const mention = slackUserId
-                ? `<@${slackUserId}>`
-                : assignee?.user_metadata?.display_name || assignee?.email || "Unknown";
-              const timeSinceLastReply = formatDuration(conversation.lastUserEmailCreatedAt!);
-              return `• <${getBaseUrl()}/mailboxes/${mailbox.slug}/conversations?id=${conversation.slug}|${subject?.replace(/\|<>/g, "") ?? "No subject"}> (Assigned to ${mention}, ${timeSinceLastReply} since last reply)`;
-            }),
-            ...(overdueAssignedConversations.length > 10
-              ? [`(and ${overdueAssignedConversations.length - 10} more)`]
-              : []),
-          ].join("\n"),
-        },
+      const blocks: KnownBlock[] = [
```

ESLint

[error] Delete `··`

(prettier/prettier)